lib/graphql/apollo-wrapper.tsx

```diff
@@ -1,7 +1,7 @@
 "use client"
 
 import { createHttpLink } from "@apollo/client";
-import { errorLink, authLink, retryLink } from "@/lib/graphql/graphqlHelper";
+import { authLink, retryLink } from "@/lib/graphql/graphqlHelper";
```

Comment:
Removed the "errorLink" from the `apollo-wrapper` since we are using handleApolloErrors with the graphql queries and mutations, and having both didn't seem necessary.